[Windows] Pass -Wno-ignored-attributes

In file included from QtWidget.mm:23:
In file included from ./QtWidget.h:24:
In file included from ../QtGui\QtPaintDevice.h:25:
In file included from G:/msys64/mingw64/include/QtGui\QPaintDevice:1:
G:/msys64/mingw64/include/QtGui/qpaintdevice.h:116:26: warning: 'QPaintDevice::devType' redeclared inline; 'dllimport' attribute ignored [-Wignored-attributes]
inline int QPaintDevice::devType() const

--- a/configure.ac
+++ b/configure.ac
@@ -35,6 +35,8 @@ AX_CHECK_COMPILER_FLAGS(-pipe, [OBJCFLAGS="$OBJCFLAGS -pipe"])
 AX_CHECK_COMPILER_FLAGS(-fno-common, [OBJCFLAGS="$OBJCFLAGS -fno-common"])
 AX_CHECK_COMPILER_FLAGS(-Wobjc-missing-property-synthesis -Werror,
 	[OBJCFLAGS="$OBJCFLAGS -Wobjc-missing-property-synthesis"])
+AX_CHECK_COMPILER_FLAGS(-Wno-ignored-attributes -Werror,
+	[OBJCXXFLAGS="$OBJCXXFLAGS -Wno-ignored-attributes"])
 
 AC_CHECK_TOOL(AR, ar)
 AC_PROG_RANLIB

